__tests__/app.test.js

```diff
@@ -0,0 +1,31 @@
+const { categoryData, commentData, reviewData, userData } = require('../db/data/test-data/index.js');
+const db = require('../db/connection.js');
+const request = require('supertest');
+const seed = require('../db/seeds/seed.js');
+const app = require('../app.js');
+
+beforeEach(() => seed({ categoryData, commentData, reviewData, userData }));
+
+afterAll(() => db.end());
+
+describe('app', () => {
+
+    describe('GET /api/categories', () => {
+        
+        test('200: Responds with an array of category objects', () => {
+            return request(app)
+                .get('/api/categories')
+                .expect(200)
+                .then(({ body }) => {
+                    const { categories } = body;
+                    categories.forEach(category => {
```

Comment:
The forEach part of test is great, but if for any reason category is not iterable (i.e not an array) this test will pass. Because of that you should also do a test for length - That will ensure that this is an array AND that the response is the length you're expected.